app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.all
+  end
+
+  def show
+    @food = Food.find(params[:id])
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params)
+    if @food.save
+      redirect_to recipes_path
+    else
+      render :new
+    end
+  end
+
+  def destroy; end
```

Comment:
The destroy method is currently empty. If this is intentional, you may want to include a comment explaining why it's empty or remove it if not needed.
Below is a suggestion you can use if you intend to keep it;
 ```
 def destroy
    @food = Food.find(params[:id])
    @food.destroy
    redirect_to foods_path, notice: 'Food deleted successfully!'
  end
```

Reply:
Thank you